.github/workflows/rust.yml

```diff
@@ -45,11 +52,19 @@ jobs:
             - name: 📎 Run clippy
               run: cargo clippy --workspace --all-targets --all-features -- -D warnings
 
-            - name: Generate JSON schema
+            - name: 📄 Generate JSON schema
               run: cargo run --bin generate_json_schema
 
             - name: 🧪 Run tests
               run: cargo nextest run --no-fail-fast --workspace
 
+            - name: 📦 Upload failed snapshot test artifacts
```

Comment:
I checked all the past failed test and did not see any one where anything was uploaded. Did you test that?

Reply:
I tested it and it works. I force-pushed my changes to this branch so the test history is overwritten. 
Here's one of the tests: https://github.com/membraneframework/video_compositor/actions/runs/6694201788

Reply:
I see that you are removing any context from file name. and user needs to identify failed test from logs. Dropping that information does not make sense if you are not gaining anything by it. Either
- add info that identifies tests to file names
- leave current naming and use the fact that you know exact paths and change the code here to upload those 2 paths separately, so we can't avoid zip file

I would prefer the second option, but don't have strong opinions about it.

Reply:
I tested the second option and it looks like github zips individual files even if I upload them separately.
https://github.com/actions/upload-artifact/issues/3#issuecomment-620722206